e2e/internal/e2e/singularitycmd.go

```diff
@@ -109,7 +109,7 @@ type singularityCmd struct {
 	envs       []string
 	dir        string
 	privileged bool
-	subTest    bool
+	noSubTest  bool
```

Comment:
no, scratch this. The way the `RunSingularity` function works requires a name for the subtest, even if the subtest is optional. I'll change that instead so that it makes sense.

Reply:
Sounds good. Ping me when you'd like the review.

Reply:
thanks @jscook2345 	

I had to change more than I had initially expected. There are two commits in this PR now: the first one removes the "name" argument, as mentioned in the cover letter. The second one adds a "cmdPath" argument, which is the path to the singularity binary, as obtained from the E2E_CMD_PATH environment variable.